homeassistant/components/shelly/binary_sensor.py

```diff
@@ -156,7 +157,7 @@
         extra_state_attributes=lambda status, shelly: {
             "latest_stable_version": status.get("stable", {"version": ""})["version"],
             "installed_version": shelly["ver"],
-            "beta_version": status.get("beta", {"version": ""})["version"],
+            "beta_version": status.get(ATTR_BETA, {"version": ""})["version"],
```

Comment:
Since this is mapping to a dictonary in the device I think we should not use `ATTR_BETA` here (all other places are ok)

Reply:
This was reverted again by the force-push